src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/prison/UserControllerIntTest.kt

```diff
@@ -517,6 +626,105 @@ class UserControllerIntTest : IntegrationTestBase() {
     }
   }
 
+  @Nested
+  inner class CreateLinkedAdminUserError {
```

Comment:
Please discuss, but I don't think all of these error integration test cases are necessary. In the event of a remote call exception we get a WebClientResponseException back. The handler for this basically replicates the error response i.e. so the same error response is given by manage-users-api. So I think only one error test case should suffice. You also don't need to put this in a separate nested inner class.